README.md

```diff
@@ -2,9 +2,9 @@
 
 # NuPIC C++ Core Library
 
-[![Linux/OSX Build Status](https://travis-ci.org/htm-community/nupic.cpp.svg?branch=master)](https://travis-ci.org/htm-community/nupic.cpp)
-[![OSX CircleCI](https://circleci.com/gh/htm-community/nupic.cpp/tree/master.svg?style=svg)](https://circleci.com/gh/htm-community/nupic.cpp/tree/master)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/59f87and1x0ugss9/branch/master?svg=true)](https://ci.appveyor.com/project/htm-community/nupic-cpp/branch/master)
+[![Linux/OSX Build Status](https://travis-ci.org/htm-community/htm.cpp.svg?branch=master)](https://travis-ci.org/htm-community/htm.cpp)
```

Comment:
these links are broken now, so should we rename to htm.cpp?

Reply:
Will be fixed in PR #526